app/models/notifications/NotificationDetails.scala

```diff
@@ -31,7 +32,14 @@ case class NotificationDetails(contactType: Option[ContactType],
 
   val cType = ContactTypeHelper.getContactType(status, contactType, variation)
 
-  def subject = s"notifications.subject.$cType"
+  def subject(templateVersion: String) = {
```

Comment:
This is inherently awkward and risky, what happens one day when v6 arrives, by default it will get the same messages as v1..4 .

I would be tempted to code this as something like:

templateVersion match {
case "v1m0" |  "v2m0" | "v3m0" | "v4m0" => s"notifications.subject.$cType
case "v5m0" => s"notifications.subject.new.$cType
case _ => throw new Exception s"Unknown template version $templateVersion"
}

Then have a notifications.subject.new.xxx for each and every cType, not just the renewalReminders.

The benefits are: that it will fail if future templateVersions are not addressed, the code here does not need to worry about which subjects change and which do not, the presumption will also be that they will not change each version after v5 (they did not so far)

Reply:
changes done 
https://github.com/hmrc/amls-frontend/commit/dbc622a525963869c2a520ceceab6155b8a8c2dc

Reply:
Thanks, you perhaps missed the subtlety that I suggested .new in place of .v5 because in all probability .v5 is going to equally apply to later versions too.  But that is no biggy, it is fine as you have done it